AmplifyPlugins/API/AWSAPICategoryPlugin/Reachability/NetworkReachabilityNotifier.swift

```diff
@@ -60,8 +71,10 @@ class NetworkReachabilityNotifier {
             isReachable = false
         }
 
-        let reachabilityMessageUpdate = ReachabilityUpdate(isOnline: isReachable)
-        reachabilityPublisher.send(reachabilityMessageUpdate)
+        if isReachable != reachabilityPublisher.value.isOnline {
```

Comment:
what was the behavior from before? if there was no check for previous value before (`if isReachable != reachabilityPublisher.value.isOnline`) then reachability always provided an update even for connection changes from `wifi` to `cellular`, but now with this check, the reachbility won't send an update